src/main/java/com/staxrt/tutorial/model/Post.java

```diff
@@ -0,0 +1,80 @@
+package com.staxrt.tutorial.model;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "posts")
+@EntityListeners(AuditingEntityListener.class)
+public class Post {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+
+    @Column(name = "data", nullable = false)
+    private String data;
```

Comment:
Co to za data? Co tam bęðzie?

Reply:
Data = dane, wszystko to czym będzie post. Date to data data to dane. Mogę zmienić nazwę jeśli wprowadza zamęt

Reply:
Nazwa mnie nie myli. Wiem, że chodzi o jakieś dane. Tylko co to za dane? Tak z perspektywy zwykłego śmiertelnika który będzie używał pamiętnika? Jedyne dane jakie mi przychodzą do głowy to treść wpisu czyli jakiś `content`. `data` się kojarzy z jakąś strukturą danych tzn. że tam będzie jakieś dziesięć pól których struktury jeszcze nie znasz. Ja bym spróbował na razie wszystkie dane trzymać bezpośrednio w kolumnach tzn. treść posta jak z czasem dojdą jakieś inne informacji typu tagi, to dodamy kolumnę tagi itd.

Reply:
rev